crazyclock.ino

```diff
@@ -48,36 +48,44 @@ const int LCD_COLS = 16;
 const int LCD_ROWS = 2;
 RotaryEncoder encoder(PIN_IN1, PIN_IN2, RotaryEncoder::LatchMode::TWO03);
 DS3231 rtc;
+
 void setup() {
+  Serial.begin(115200);
+  while (!Serial) {
+    // waits for serial port to be ready
+  }
+  Serial.println("\n================");
+  Serial.println("crazyclock");
+  Serial.println("================");
+  Serial.println("Starting I2C...");
+  Wire.begin();
```

Comment:
I needed this line because of some troubles for multiple I2C devices and Wemos D1.

It is also suggested for use with [DS3231 rtc](https://github.com/NorthernWidget/DS3231#summary).

Reply:
Also drum roll would be appreciated ;)